internal/healthcheck/check.go

```
@@ -19,22 +19,31 @@ func (ch *CachedResponses) process(
 ) {
 	wg := sync.WaitGroup{}
 
-	service := &cfg.Cluster
-	if service.Enabled && len(service.Resources) > 0 {
+	cluster := &cfg.Cluster
+	if cluster.Enabled && len(cluster.Resources) > 0 {
+		if cluster.Tag == "" {
+			cluster.Tag = "cluster"
+		}
 		wg.Go(func() {
```

golangci-lint (2.11.3)

[error] missing whitespace above this line (invalid statement above expr)

(wsl_v5)